utils/gatsby-theme.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { createRef } from 'react';
+import { ThemeProvider, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+export const wrapRootElement = ({ element }) => {
+  const focusVisibleRef = createRef(null);
+
+  const theme = {
+    ...DEFAULT_THEME,
+    palette: { ...DEFAULT_THEME.palette, tofu: '#F6F4F4', oatMilk: '#EDE0CF' }
+  };
+
+  return (
+    <ThemeProvider focus={focusVisibleRef} theme={theme}>
+      <div
+        ref={focusVisibleRef}
+        css={`
+          width: 100%;
+          height: 100%;
+        `}
```

Comment:
This makes me wonder if we  have a forwardRef or prop spreading problem on Garden's `ThemeProvider`.

Reply:
This is a unique scenario due to our `focus-visible` hook. Some more traditional ways would be to apply styling to the parent of this `<div>`, but the Gatsby abstractions make this difficult to do. For a majority of consumers (`create-react-app` based) it should be easier.